server/container/guice/cassandra-guice/src/test/java/org/apache/james/DockerCassandraRule.java

```diff
@@ -41,7 +41,7 @@ private static boolean isBindingToEveryThing(Binding binding) {
         return bindingIp == null || bindingIp.equals("0.0.0.0");
     }
 
-    private PropertiesConfiguration getCassandraConfigurationForDocker() {
+    public PropertiesConfiguration getCassandraConfigurationForDocker() {
```

Comment:
I would like to have find a elegant solution in Guice so I can "decorate" a module. By that I mean retrieve the original binding for a class modify it and bind the modified instance but I could not find how to do that.